src/web/components/ArticleHeadline.tsx

```diff
@@ -1,13 +1,11 @@
 import React from 'react';
 import { css, cx } from 'emotion';
 
-import ClockIcon from '@frontend/static/icons/clock.svg';
 import { getAgeWarning } from '@root/src/lib/age-warning';
 
-import { palette } from '@guardian/src-foundations';
-import { headline, textSans } from '@guardian/src-foundations/typography';
+import { AgeWarning } from '@root/src/web/components/AgeWarning';
```

Comment:
I think `@frontend/` rather than `@root`?

Reply:
At some point Visual Code got all opinionated and we lost a lot of `@frontend` and gained a load of `@root/src`.

Shall we do a big PR to move all of these over? Arguably, we could lose `root` completely as with workspaces removed it's a bit redundant

Reply:
Yup, makes sense to do them all in one 👍